CHANGES.rst

```diff
@@ -568,7 +574,7 @@ parameter as described below:
 
 * Uploading an image resulting in too large of a request (>500 KB) now
   raises ``prawcore.TooLarge`` instead of an ``AssertionError``.
-* Uploading an invalid image raises :class:`.APIException`.
+* Uploading an invalid image raises :class:`.RedditAPIException`.
```

Comment:
Let's avoid updating old error log entries, other than to fix errors from sphinx not being able to link when something has been deleted.

Reply:
This is why I did the update, aliases aren't recognized as classes.